src/DssSpell.t.sol

```diff
@@ -899,205 +868,46 @@ contract DssSpellTest is DssSpellTestBase {
-        GodMode.setBalance(address(dai), address(pair), daiAmt);
-        uint256 mkrAmt = 1_000_000 * WAD * WAD / price;
-        GodMode.setBalance(address(gov), address(pair), mkrAmt * 97 / 100); // 3% worse price (should fail)
-        vm.expectRevert("FlapperUniV2/insufficient-buy-amount");
-        vow.flap();
-        GodMode.setBalance(address(gov), address(pair), mkrAmt * 99 / 100); // Leaves just 1% worse price
-        //
-
-        uint256 initialLp = GemAbstract(pair).balanceOf(pauseProxy);
-        uint256 initialDaiVow = vat.dai(address(vow));
-        uint256 initialReserveDai = dai.balanceOf(pair);
-        uint256 initialReserveMkr = gov.balanceOf(pair);
-
-        vow.flap();
-
-        assertGt(GemAbstract(pair).balanceOf(pauseProxy), initialLp);
-        assertGt(dai.balanceOf(pair), initialReserveDai);
-        assertEq(gov.balanceOf(pair), initialReserveMkr);
-        assertGt(initialDaiVow - vat.dai(address(vow)), 2 * vow.bump() * 9 / 10);
-        assertLt(initialDaiVow - vat.dai(address(vow)), 2 * vow.bump() * 11 / 10);
-        assertEq(dai.balanceOf(address(flap)), 0);
-        assertEq(gov.balanceOf(address(flap)), 0);
-
-        // Check Mom can increase hop
-        assertEq(flap.hop(), 1577 seconds);
-        vm.prank(chief.hat());
-        flapMom.stop();
-        assertEq(flap.hop(), type(uint256).max);
+        _checkRWADocUpdate("RWA002-A", OLD_RWA002_DOC, NEW_RWA002_DOC);
     }
 
-    function testSequencerFlapJob() public {
-        DssCronSequencerLike sequencer = DssCronSequencerLike(addr.addr("CRON_SEQUENCER"));
-        DssCronJobLike flapJob = DssCronJobLike(addr.addr("CRON_FLAP_JOB"));
-
-        assertTrue(!sequencer.hasJob(address(flapJob)));
-
-        // execute spell
+    function testRWA004OracleTell() public {
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        assertTrue(sequencer.hasJob(address(flapJob)));
-
-        address pip = flap.pip();
-        address pair = flap.pair();
-
-        vm.prank(pauseProxy);
-        MedianAbstract(pip).kiss(address(this));
-
-        uint256 price = MedianAbstract(pip).read();
-        uint256 daiAmt = 1_000_000 * WAD;
-        GodMode.setBalance(address(dai), address(pair), daiAmt);
-        uint256 mkrAmt = 1_000_000 * WAD * WAD / price;
-        GodMode.setBalance(address(gov), address(pair), mkrAmt * 99 / 100); // Leaves just 1% worse price
+        (, , uint tau, uint toc) = oracle.ilks("RWA004-A");
+        assertGt(toc, 0, "RWA004-A: bad `toc` after `tell()`");
 
-        bytes32 master = sequencer.getMaster();
-        uint256 snapshot = vm.snapshot();
-        (bool ok, bytes memory data) = flapJob.workable(master);
-        vm.revertTo(snapshot);
-        assertTrue(ok);
-        flapJob.work(sequencer.getMaster(), data);
+        skip(tau);
+        assertEq(oracle.good("RWA004-A"), false, "RWA004-A: still `good` after `tell()` + `tau`");
```

Comment:
I agree with this though I won't mandate it / mark it as blocking (personally) as this is a novel action and it's the first time this is happening (and this is an RWA ilk, so much of this is paper-shifting so to speak).

Reply:
Guys, the TinlakeMgr IS NOT MakerDAO code. It's 3rd party and this sort of test should've been covered in the development phase of the TinlakeMgr component itself.

We shouldn't be checking the integration works as intended is spells.

Reply:
> Guys, the TinlakeMgr IS NOT MakerDAO code. It's 3rd party and this sort of test should've been covered in the development phase of the TinlakeMgr component itself.

True

> We shouldn't be checking the integration works as intended is spells.

not sure what it means. Should "is" be "in"? I will assume is should. Here i would respond:

We rather don't have because it's not required to do so but should do this because the result of the spell enables their action. It's sort of an E2E test.

---

Taking into account that:
  - it's not marked as blocking by the second reviewer (most importantly :) );
  - i've played around with this test and extended it myself - nothing broke;
  - technically (as per raised points) it's not required by the process - i'm not entitled to block the progress / veto-ing would be abuse;

... - i see that further conversation will not result into anything productive because it's "extra-step-make-sure" vs declared processes. 

So if you don't feel like extending the test - there's no damage afaik. Let's drop this here in such a case.